specification/amqp_0.9.1/src/main/scripts/org/kaazing/specification/amqp/tcp/open/noidentity/request.rpt

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#connect tcp://127.0.0.1:8011
```

Comment:
If I include tcp.connect.rpt into request.rpt I have to add it to 2 different scripts (Not a big overhead but current script structure feels more comfortabele)